string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,30 @@
+/* exported numVowels */
+// create a counter variable
+// capitalize the whole string put in new variable.
+// write a loop that loops through the new string
+// in the loop write an if statement that compares string[i] to vowels A E I O U
+// if comparison is true increment the counter
+// return counter
+
+function numVowels(string) {
+  var counter = 0;
+  var string2 = string.toUpperCase();
```

Comment:
Numbered variable names are The Devil. Instead of making a new variable here, you should just reassign to `string`. 
```
string = string.toUpperCase();
```